toqito/states/__init__.py

```diff
@@ -1,4 +1,7 @@
-"""This module refers to the `Quantum States` which acts as the parent to the below submodules. Quantum states are one of the three fundamental objects that `toqito` provides, the others being `channels` and `measurements`."""
+"""Quantum States is used to denote the module `Quantum State`. It acts as the parent to the below submodules.
```

Comment:
This isn't entirely true; there is no module called "Quantum State." Instead, this is a set of modules that numerically implement well-known quantum states in the literature.

Reply:
Ohh okay. I will correct it.
Thank you